spyder/plugins/ipythonconsole/utils/kernelspec.py

```diff
@@ -186,7 +190,7 @@ def env(self):
 
         # Environment variables that we need to pass to the kernel
         env_vars.update({
-            'SPY_EXTERNAL_INTERPRETER': not default_interpreter,
+            'SPY_EXTERNAL_INTERPRETER': not default_interpreter or self.path_to_custom_interpreter,
```

Comment:
```suggestion
            'SPY_EXTERNAL_INTERPRETER': (not default_interpreter
                or self.path_to_custom_interpreter),
```

This makes this line fit in 79 columns.